keyboards/matrimony/readme.md

```diff
@@ -0,0 +1,9 @@
+# Matrimony
+
+* Keyboard Maintainer: [Joshua Cheung](https://github.com/hypanim)
+* Hardware Supported: Elite-C
+* Hardware Availability: https://www.reddit.com/r/mechmarket/comments/w1ytmc/gb_matrimony_split_keyboard/?utm_source=share&utm_medium=web2x&context=3
+
+Both sides of the split keyboard are flashed with the same firmware. Make sure to set the eeprom of each side to their respective left/right configuration using QMK toolbox before flashing. The BootMagic key for the left side is the top left, while the BootMagic key for the right side is the top right. 
+```
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Please add instructions on entering bootloader mode. 
eg.
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available

Reply:
Thank you so much for your help! I'm currently putting all these changes together.